### ISSUE TITLE
igraph depends on BLAS

### ISSUE THREAD

**Issue by tueda** [Linuxbrew/homebrew-core#4992]
- [x] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Linuxbrew/brew/issues/new.
- [ ] If you have a macOS system and can replicate the problem on macOS with Homebrew, please report the issue to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and follow their issue template from your macOS system. If you are unsure, please report the issue to Linuxbrew.
- [x] Ran `brew update` and retried your prior step?
- [ ] Ran `brew doctor`, fixed all issues and retried your prior step?
- [ ] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

---

Currently `igraph` (now in `homebrew-core`, migrated from `homebrew-science`) can't be built from source on Linux without manually installing `openblas`:
```
/localstore/theorie/tueda/build/linuxbrew/bin/ld: cannot find -lblas
/localstore/theorie/tueda/build/linuxbrew/bin/ld: cannot find -llapack
```
 It seems that the patch Homebrew/homebrew-science#6038 must be reapplied. (Maybe only in Linuxbrew? Or in the upstream?)

**Comment by iMichka** [comment]
Hi. Yes, these changes need to be done. The two dependencies need to be added within an `unless OS.mac?  / end` block.

Would you be willing to open a pull request in this repository with a fix for this?

**Comment by tueda** [comment]
OK. I will try for that.

**Comment by iMichka** [comment]
Fixed by #4993
